src/cim/components/Agent/AgentsSelectionHostCountAlerts.tsx

```diff
@@ -30,9 +31,10 @@ const AgentsSelectionHostCountAlerts = ({
       {selectedAgentsCount === targetHostCount && (
         <Alert
           variant={AlertVariant.success}
-          title={`${selectedAgentsCount} ${
-            selectedAgentsCount === 1 ? 'host' : 'hosts'
-          } selected out of ${matchingAgentsCount} matching.`}
+          title={i18next.t(
```

Comment:
by https://github.com/openshift-assisted/assisted-ui-lib/pull/1399#discussion_r883537804
I meant that this component should use hook to get the `t`
You changed it to static access which is a bit better but still..

```suggestion
const { t } = useTranslation();
...
...
          title={t(
```